### PR TITLE
fix: Add missing queries to fusion input adapter

### PR DESCRIPTION
**Pull request by bananasov**
Added water, tritium, deuterium and DT Fuel

I wanted to look at fission and the rest from #73 but apparently they aren't missing those inputs, fusion did though

### CODE REVIEW

**Comment by bananasov** [comment]
Apparently, evaporation towers are missing input and outputs too

File: src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua [hunk 1]
```diff
@@ -36,16 +36,21 @@ function FusionReactorInputAdapter:beforeRegister ()
...
         coolant = {
+            water                       = fn():call('getWater'):div(1000):fluid(),
             water_capacity              = fn():call('getWaterCapacity'):div(1000):fluid(),
             water_needed                = fn():call('getWaterNeeded'):div(1000):fluid(),
+            steam                       = fn():call('getSteam'):div(1000):fluid(),
             steam_capacity              = fn():call('getSteamCapacity'):div(1000):fluid(),
             steam_needed                = fn():call('getSteamNeeded'):div(1000):fluid(),
         },
         fuel = {
+            tritium                     = fn():call('getTrititum'):div(1000):fluid(),
```
Comment (cyberbit) [inline src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua:47]: Not sure if it is a typo in the mod or here, but I will check in-game later on if `getTrititum` is right. Wouldn't be the first time something like that is real.

Comment (cyberbit) [reply src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua:47]: Looks like it is `getTritium`:

image

File: src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua [hunk 2]
```diff
@@ -36,16 +36,21 @@ function FusionReactorInputAdapter:beforeRegister ()
         --     environmental_loss       = fn():call('getEnvironmentalLoss'),
         -- },
         coolant = {
+            water                       = fn():call('getWater'):div(1000):fluid(),
```
Comment (cyberbit) [inline src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua:39]: These need to be modified to pull `amount` from the result of the call before calculating. See example from other adapters below:

https://github.com/cyberbit/telem/blob/f7e54a8d6cb496040cb7d15f99a7a0231a409fab/src/telem/lib/input/mekanism/PressurizedReactionChamberInputAdapter.lua#L19

File: src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua (hunk 1 above)
Comment (cyberbit) [inline src/telem/lib/input/mekanism/FusionReactorInputAdapter.lua:47]: The call has a typo, should be `getTritium`

**Review by cyberbit — CHANGES_REQUESTED** [review]
Looks good, if you can fix typo and the `amount` things on the additions I will merge!

**Review by cyberbit — APPROVED** [review]

**Pull request merged**